docs/built-ins/suspense.md

```diff
@@ -1,16 +1,12 @@
----
-outline: deep
----
+# Suspense
 
-# Suspense {#suspense}
 
-:::warning Experimental Feature
 `<Suspense>` is an experimental feature. It is not guaranteed to reach stable status and the API may change before it does.
```

Comment:
Maybe you can use the warning box here so its more similar to the original page?